postal/service_test.go

```diff
@@ -714,258 +716,6 @@ version = "this is super not semver"
 		})
 	})
 
-	context("Install", func() {
```

Comment:
At a glance, these looked to be covered by the `"Deliver"` context tests. Feel free to correct me if I missed something.

Reply:
Yeah. This is fine. I always figured they just get removed when we removed the method.